streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -27,7 +27,7 @@
 
 public interface Task {
     /**
-     * Initialize the task and return {}true if the task is ready to run, i.e, it has not state stores
```

Comment:
I think the original intent was: `...return {@code true} if the...`

Reply:
+1. cc @kamalcph